url64b10c0106_schlabberdog_blocks_tgz-pJ8-com_github_users_schlabberdog_blocks_ui_CliUIJ8/src/com/github/users/schlabberdog/blocks/r010/RLBlock.java

```diff
@@ -0,0 +1,78 @@
+package com.github.users.schlabberdog.blocks.r010;
+
+import com.github.users.schlabberdog.blocks.board.Block;
+import com.github.users.schlabberdog.blocks.board.Board;
+import com.github.users.schlabberdog.blocks.board.moves.IMove;
+import com.github.users.schlabberdog.blocks.board.moves.Move;
+import com.github.users.schlabberdog.blocks.mccs.Coord;
+import com.github.users.schlabberdog.blocks.mccs.Rect;
+
+import java.awt.*;
+import java.util.ArrayList;
+
+/* |XX
+ * | X
+ */
+public class RLBlock extends Block {
+
+    public RLBlock() {
+        super(2,2);
+    }
+
+
+    @Override
+    public void printOntoMap(Coord pos,char[][] m) {
+        char c = getRepresentation();
+        m[pos.y][pos.x] = c;
+        m[pos.y][pos.x+1] = c;
+        m[pos.y+1][pos.x+1] = c;
+    }
+
+    @Override
+    public void addAlts(Coord pos, Board board, ArrayList<IMove> alts) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 11. By comparison, 98% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.